### PR TITLE
Rmd file

### PR DESCRIPTION
**Pull request by vgb2113**

### CODE REVIEW

File: EDAVSurveyAndAnalysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# EDAV Survey and Analysis
+Varalika Mahajan and Vrinda Bhat
```
Comment (philkim72) [inline EDAVSurveyAndAnalysis.Rmd:2]: Per the instructions, The second line should be blank, followed by the author name(s).

**Review by philkim72 — CHANGES_REQUESTED** [review]
Per the instructions, the .Rmd filename can only contain lowercase letters.

**Review by vgb2113 — COMMENTED** [review]
Modified the file as per instructions.

**Comment by philkim72** [comment]
Thank you for your contribution :tada:

**Pull request merged**